code/src/App.js

```diff
@@ -1,12 +1,12 @@
-import React from 'react'
-import data from './data.json'
-
-console.log(data)
+import React from 'react';
+import data from './data.json';
+import { AlbumsContainer } from './Components/AlbumsContainer';
```

Comment:
You don't need brackets when you are importing one thing
```suggestion
import AlbumsContainer from './Components/AlbumsContainer';
```